gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/pooled_grpc_asyncio.py.j2

```diff
@@ -0,0 +1,445 @@
+                ],
+            )
+        old_channel = self.channel_pool[channel_idx]
+        self.channel_pool[channel_idx] = new_channel
+        await old_channel.close(grace=grace)
+        # invalidate stubs
+        stub_keys = list(self._stubs.keys())
+        for stub_channel, stub_func in stub_keys:
+            if stub_channel == old_channel:
+                del self._stubs[(stub_channel, stub_func)]
+        return new_channel
+
+    {% if service.has_lro %}
+
+    @property
+    def operations_client(self) -> operations_v1.OperationsAsyncClient:
+        """Create the client designed to process long-running operations.
+
+        This property caches on the instance; repeated calls return the same
+        client.
+        """
+        # Quick check: Only create a new client if we do not already have one.
+        if self._operations_client is None:
+            self._operations_client = operations_v1.OperationsAsyncClient(
+                self.grpc_channel
+            )
+
+        # Return the client from cache.
+        return self._operations_client
+    {% endif %}
+    {% for method in service.methods.values() %}
+
+    def {{ method.transport_safe_name|snake_case }}(self, *args, **kwargs) -> Awaitable[{{ method.output.ident }}]:
+        r"""Function for calling the{{ ' ' }}
+        {{- (method.name|snake_case).replace('_',' ')|wrap(
+                width=70, offset=40, indent=8) -}}
+        {{ ' ' }}method over gRPC.
+
+        {{ method.meta.doc|rst(width=72, indent=8) }}
+
+        Returns:
+            Callable[[~.{{ method.input.name }}],
+                    Awaitable[~.{{ method.output.name }}]]:
+                A function that, when called, will call the underlying RPC
+                on the server.
+        """
+        # Generate a "stub function" on-the-fly which will actually make
+        # the request.
+        # gRPC handles serialization and deserialization, so we just need
+        # to pass in the functions for each.
+        next_channel = self.next_channel()
+        stub_key = (next_channel, '{{ method.transport_safe_name|snake_case }}')
+        stub_func = self._stubs.get(stub_key, None)
+        if stub_func is None:
+            stub_func = next_channel.{{ method.grpc_stub_type }}(
+                '/{{ '.'.join(method.meta.address.package) }}.{{ service.name }}/{{ method.name }}',
+                request_serializer={{ method.input.ident }}.{% if method.input.ident.python_import.module.endswith('_pb2') %}SerializeToString{% else %}serialize{% endif %},
+                response_deserializer={{ method.output.ident }}.{% if method.output.ident.python_import.module.endswith('_pb2') %}FromString{% else %}deserialize{% endif %},
+            )
+            self._stubs[stub_key] = stub_func
+        # call stub
+        return stub_func(*args, **kwargs)
+    {% endfor %}
+
+    {# TODO: Remove after https://github.com/googleapis/gapic-generator-python/pull/1240 is merged. #}
```

Comment:
Yeah, for now I'm leaving it in place because (last I checked) this is still present in the other transport, and I wanted to minimize the diff during development. but I will make sure this is fixed before merging